posts/make-your-react-native-app-lighter-and-faster/2019-11-14-make-your-react-native-app-lighter-and-faster.md

```diff
@@ -0,0 +1,63 @@
+---
+templateKey: 'blog-post'
+title: 'Make your React Native app lighter and faster'
+date: 2019-11-14
+featuredpost: false
+description: >-
+  Optimzations for smaller, lighter, faster, smoother react native app.
+keywords:
+- react native
+- react native app
+- optimze
+- fast
+- small
+- smooth
+link: /make-your-react-native-app-lighter-and-faster
+category:
+- Technology
+tags:
+- react-native
+- mobile-app-development
+author: Arivanandan
+---
+
+# Make your React Native app lighter and faster
```

Comment:
You can delete this line. Blog title will be generated from the `frontmatter`. Currently it's displaying two titles.